guides/centering_accuracy.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: guides
+title: Guides | Centring Accuracy 
+description: This guide teaches you how to centre the accuracy on the playfield.
+---
+
+# Centring Accuracy
+
+This guide will teach you how to centre the accuracy counter on the playfield.
+
+Note that by doing this, the percentage symbol after the accuracy will disappear. This is due to osu! stopping to load the texture and just the hitbox, due to its size.

```

Comment:
"and just the hitbox" sounds like a verb is missing here.
Would go
This is due to osu! stopping to load the texture and it only loads the hitbox, due to its size.